pyscitt/pyscitt/receipt.py

```diff
@@ -88,7 +88,7 @@ def cbor_to_printable(cbor_obj: Any, cbor_obj_key: Any = None) -> Any:
         if cbor_obj_key.identifier == X5chain.identifier:
             return [base64.b64encode(cert).decode("ascii") for cert in cbor_obj]
         if cbor_obj_key.identifier == KID.identifier:
-            return cbor_obj.decode()
+            return cbor_obj.hex()
```

Comment:
The method is used to pretty-print the cbor structure, we had KID bytes representing a string and decoding seemed to pretty print it well, why change to hex? Did kid change to something which is not a string anymore?

Reply:
kids for CCF COSE signatures are SHA-256, they are not strings.